Dockerfile

```diff
@@ -1,8 +1,12 @@
 # syntax=docker/dockerfile:1
 
+# ubuntu base is only used for riscv64 builds. we also need to keep debian to
+# be able to build against armel arch
+# TODO: remove ubuntu base when riscv64 is supported by golang debian base image
+ARG DEBIAN_BASE="debian:bullseye"
+ARG UBUNTU_BASE="ubuntu:22.04"
```

Comment:
I'd like to block this new target on Debian support instead of introducing Ubuntu.

Reply:
Unfortunately debian doesn't provide any cross pkg for riscv64 architecture (even sid). Also the ubuntu image is just used for the riscv64 arch.

Reply:
https://lwn.net/Articles/939095/

Good news :)

Reply:
Just in time!